typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -337,12 +415,29 @@ export function updateTokenBalanceMetrics(
         token_name: token.name,
         wallet_address: token.hypAddress,
         token_type: token.type,
+        warp_route_id: createWarpRouteConfigId(
```

Comment:
@tkporter - we discussed the limitations of using this, mainly being that it can change for the renzo case and we would need to change to config infra every time and that it is very long and increasing in length

How about we have manual override in our infra config that could be used instead if set? We can set this manual override for renzo only for now?

Reply:
@tkporter!